src/pudl/transform/epacems.py

```diff
@@ -208,9 +230,9 @@ def transform(raw_df: pd.DataFrame, pudl_engine: sa.engine.Engine) -> pd.DataFra
     """
     return (
         raw_df.fillna({"gross_load_mw": 0.0, "heat_content_mmbtu": 0.0})
```

Comment:
Why are we doing this? To me, NA represents a lack of data where 0 represents a reading of 0. I don't think changing these values to 0 is accurate.

Reply:
I can do some more investigation, but my guess is that there was either an error with the CEMS technology or there simply wasn't a reading. It doesn't make sense that load or head content would be zero if the plant was actually functioning.

Reply:
@grgmiller do you have any thoughts on this?

Reply:
Thanks for flagging this. I would strongly recommend that NA values should definitely be left as NA, and the end user of the data can choose to fill the NA if they want. It seems like the current practice of filling NAs with zeros would contradict the "[minimize data alteration](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/data_guidelines.html#minimize-data-alteration)" principle, since you are essentially adding new data points where they do not exist in the original data.

One example of why this is important: in our hourly eGRID work, we are combining data from CEMS and EIA-923 about the hourly generation and fuel data from plants. If measured data from CEMS is available, we always prioritize it over EIA-923 data, but if CEMS is missing data, we revert to using EIA-923 data.

So in this case, if pudl is filling missing values with zero, we would be treating those zero values as measured zeros in CEMS, and overriding data from EIA-923, rather than dropping the NA records from CEMS so that we know to use EIA-923 data for that generator in that case.